aten/src/ATen/core/LegacyTypeDispatch.h

```diff
@@ -20,26 +20,26 @@ namespace at {
 
 class CAFFE2_API LegacyTypeDispatch {
```

Comment:
Probably an even further improvement (which is not for this PR) would be to eliminate the dynamic dispatch indirection, now that the initializations are being done from code that has direct access to it (i.e., is not from `torch_cpu`)